salt/metalk8s/kubeadm/init/certs/ca.sls

```diff
@@ -0,0 +1,45 @@
+{%- from "metalk8s/map.jinja" import ca with context %}
+
+Create CA salt signing_policies:
+  file.serialize:
+    - name: /etc/salt/minion.d/30-metalk8s_ca_signing_policies.conf
+    - user: root
+    - group: root
+    - mode: 644
+    - makedirs: True
+    - dir_mode: 750
+    - formatter: yaml
+    - dataset: {{ {"x509_signing_policies": ca.signing_policies} }}
+
+Restart salt-minion:
+  cmd.run:
+    - name: 'salt-call --local service.restart salt-minion'
+    - bg: true
+    - onchanges:
+      - file: Create CA salt signing_policies
+
+Create CA private key:
+  x509.private_key_managed:
+    - name: {{ ca.cert.signing_private_key }}
+    - mode: 600
+    - makedirs: True
+
+Generate CA certificate:
+  x509.certificate_managed:
+    {%- for key, value in ca.cert.items() %}
+    - {{ key }}: {{ value }}
+    {%- endfor %}
+    - makedirs: True
+    - require:
+      - x509: Create CA private key
+
+# TODO: Find a better to way to advetise CA server
```

Comment:
little typo: an extra 'to' after the better

Reply:
Oh :(